web/annotation_layer_builder.css

```diff
@@ -93,6 +93,60 @@
   width: 115%;
 }
 
+.annotationLayer .checkboxWidgetAnnotation div,
+.annotationLayer .radioButtonWidgetAnnotation div {
+  background-color: rgba(0, 54, 255, 0.13);
```

Comment:
That's the color used by you guys in `.textWidgetAnnotation`. It looks pretty much like Adobe Reader.